EncoreTickets.SDK.Tests/UnitTests/Inventory/InventoryServiceApiTests.cs

```diff
@@ -245,7 +246,7 @@ public void GetAvailabilities_IfProductIdIsSet_CallsApiWithRightParameters(strin
             AssertExtension.AreObjectsValuesEqual(expected, actual);
         }
 
-        [TestCaseSource(typeof(InventoryServiceApiTestsSource), nameof(InventoryServiceApiTestsSource.GetAvailabilities_IfApiResponseFailed_ThrowsApiException))]
+        [TestCaseSource(typeof(InventoryServiceApiTestsSource), nameof(InventoryServiceApiTestsSource.Availabilities_IfApiResponseFailed_ThrowsApiException))]
```

Comment:
The same thing with the property name, as we discussed earlier. Please check all modified files, I see this renaming in many classes. And it looks like only properties have been renamed that start with “Get”